tools/components/admin/generate-report-steps.ts

```diff
@@ -0,0 +1,4 @@
+export enum GenerateReportModalStep {
+  selectColumns = 1,
+  export = 2,
+}
```

Comment:
This file shouldn't be here - make sure to delete it

Reply:
oh whoops gotcha